lib/MadNLPGPU/Project.toml

```diff
@@ -14,7 +14,7 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
 AMD = "0.5"
-CUDA = "5.4.0"
+CUDA = "5.4.0 - 5.8.2"
```

Comment:
This is useless done here only, the only result will be that older versions of this package will be installed together with newer CUDA.jl.  You want to fix compat bounds in the registry as well.